### PR TITLE
Replace main/perl-try-tiny with newer community/perl-try-tiny

### PR DESCRIPTION
**Pull request by J0WI**
There are some more duplicated pkgs:
```
      2 ortp
      2 perl-class-load
      2 perl-cpan-meta-check
      2 perl-data-optlist
      2 perl-dist-checkconflicts
      2 perl-module-implementation
      2 perl-module-runtime
      2 perl-sub-exporter
      2 perl-sub-install
      2 perl-test-without-module
```

Maybe it's worth to implement a check for this.

### CODE REVIEW

**Comment by J0WI** [comment]
Some other trivial duplicates in `testing`:
```
      2 haveged
      2 lua-uuid
      2 perl-net-cidr-lite
      2 perl-scope-guard
      2 perl-specio
      2 py-py
```
```
      2 gdnsd
      2 geocode-glib
      2 libplist
      2 perl-data-compare
      2 perl-data-dump
```

**Comment by J0WI** [comment]
@andypost do you know how to proceed with duplicated packages?

**Comment by andypost** [comment]
@JOWI I'd better filed issue to bugs.a.o and collected this duplicates and links to PRs there, so all this discussions could be found in one place

File: main/perl-try-tiny/APKBUILD [hunk 1]
```diff
@@ -1,36 +1,38 @@
 # Automatically generated by apkbuild-cpan, template 1
-# Contributor: Kiyoshi Aman <kiyoshi.aman@gmail.com>
-# Maintainer: Kiyoshi Aman <kiyoshi.aman@gmail.com>
```
Comment (ncopa) [inline main/perl-try-tiny/APKBUILD]: I think it would make sense to keep at least the Contributor comment.

Comment (J0WI) [reply main/perl-try-tiny/APKBUILD]: Done

Comment (J0WI) [reply main/perl-try-tiny/APKBUILD]: :ping_pong: @ncopa

**Comment by maxice8** [comment]
@fcolista can be closed.

Sorry this was fixed on a PR i made later but i didn't see this one was already made.

**Pull request closed without merging**